website/docs/components/toast/partials/guidelines/overview.md

```diff
@@ -1 +1 @@
-A Toast is an element intended for messages resulting from a user’s actions. It typically disappears after a set interval or due to a user dismissing it. For system-generated messages, see the [Alert](/components/alert) component.
\ No newline at end of file
+Toasts deliver messages to communicate an ongoing process or result of a user action. They typically disappear after a set interval or upon user dismissal, offering a non-intrusive way to provide feedback. 
```

Comment:
Slight change to communicate that toasts may not always be the right component to communicate a result of a user action. wdyt?

```suggestion
Toasts deliver messages to communicate an ongoing process or, sometimes, a result of a user action. They typically disappear after a set interval or upon user dismissal, offering a non-intrusive way to provide feedback. 
```